controllers/datadogagent/feature/tcpqueuelength/feature.go

```diff
@@ -69,7 +75,7 @@ func (f *tcpQueueLengthFeature) ConfigureV1(dda *v1alpha1.DatadogAgent) (reqComp
 // ManageDependencies allows a feature to manage its dependencies.
 // Feature's dependencies should be added in the store.
 func (f *tcpQueueLengthFeature) ManageDependencies(managers feature.ResourceManagers, components feature.RequiredComponents) error {
-	return nil
+	return managers.ConfigMapManager().AddConfigMap(component.GetDefaultSecCompConfigMapName(f.owner), f.owner.GetNamespace(), agent.DefaultSecCompConfigDataForSystemProbe())
```

Comment:
is the same configmap required for all system probe features? if so, should we add it to `enabledefault` after checking if system probe is enabled?